### PR TITLE
[PLATFORM-1322]: Replace pyxis with another css lib

### PR DESCRIPTION
**Pull request by cottinisimone**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/PLATFORM-1322

The final result is:
image

### CODE REVIEW

File: web/src/pages/home/mod.rs [hunk 1]
```diff
@@ -116,49 +116,99 @@ impl Component for Home {
 
                 true
             }
+            Msg::PermissionKeyUp(event) => {
+                if 13 == event.key_code() {
```
Comment (cpiemontese) [inline web/src/pages/home/mod.rs:120]: I'd make `13` a `const` with a meaningful name

**Review by cpiemontese — COMMENTED** [review]
Just one comment for now, I still want to see what it looks like locally

Also :+1: for Bulma

File: web/src/pages/home/mod.rs (hunk 1 above)
Comment (cottinisimone) [reply web/src/pages/home/mod.rs:120]: Good point

**Review by cpiemontese — APPROVED** [review]
Really nice! It kinda breaks down on smaller screen but I doubt anyone uses it on such small screens.

image

**Review by MaeIsBad — APPROVED** [review]
nice!

**Pull request merged**